node/index.js

```diff
@@ -0,0 +1,32 @@
+const { Pool } = require('pg')
+
+let client
+let pool
+
+const initTable = async (client) => {
+  console.log('Initializing table...')
+  const tableInitialized = await client.query('CREATE TABLE IF NOT EXISTS accounts (id UUID PRIMARY KEY DEFAULT gen_random_uuid(), balance INT8)')
+  console.log(tableInitialized)
+}
+
+const insertAccounts = async (client) => {
+  console.log('Inserting new rows into the accounts table...')
+  const balanceValue = [Math.floor(Math.random() * 1000)]
+  const accountsInserted = await client.query('INSERT INTO accounts (id, balance) VALUES (DEFAULT, $1)', balanceValue)
+  console.log(accountsInserted)
+}
+
+exports.handler = async (event, context) => {
```

Comment:
We don't seem to do anything different based on the "event".

Wondering if we should be differentiating here between the different event types, and potentially including some graceful shutdown logic.